### ISSUE TITLE
The app is not loading.

### ISSUE THREAD

**Issue by TurbooGhost** [aichingm/lonewolf#2]
Hi @aichingm 

OS: Debian GNU/Linux 12 (bookworm) x86_64
Kernel: 6.1.0-12-amd64
GNOME 43.6
node -v: v18.13.0

Thanks for the great work guys

![273076660-4e1165ec-9705-4945-b296-e56eb8f88827](https://github.com/aichingm/lonewolf/assets/61920916/03f17761-c64f-4fbf-9435-21816bdfb3f1)

**Comment by Terkyz** [comment]
Same thing on KDE Neon.

**Comment by aichingm** [comment]
hi @TurbooGhost, you closed the issue. Was this done on purpose? If so could you just leave a sentence or two describing how you ware able to fix this issue?

hi @Terkyz, are you still experiencing this issue?

**Comment by Terkyz** [comment]
Yes, I'm still experiencing it, I though were you who closed the issue.

**Comment by TurbooGhost** [comment]
Hi @aichingm 

Didn't mean to close.

**Comment by aichingm** [comment]
@TurbooGhost @Terkyz I tried to figure this one out but I can't reproduce it. I could use your help here!

Could you post the output of:

```
flatpak info site.someones.Lonewolf
```

```
flatpak info --show-permissions site.someones.Lonewolf
```

```
flatpak run site.someones.Lonewolf
```

And last but not least, a really good source of information would be if you could provide me with the in-app console output:

1. Right click on the window 
![image](https://github.com/aichingm/lonewolf/assets/6470959/dce618d3-77e8-48ac-9b8f-3fc3f139ea84)
1. Click on "Inspect Element"
1. Click on the "Console" tab (red) and send me everything in the blue rectangle 
![image](https://github.com/aichingm/lonewolf/assets/6470959/8325e3c7-fbeb-4630-961c-189fd6fdd9a2)

Thanks for you patience!

**Comment by Terkyz** [comment]
Sure, no problem.
```
terkyz@terkyz-linux:~$ flatpak info site.someones.Lonewolf

Lonewolf - Organize and track your tasks with ease and flexibility

          ID: site.someones.Lonewolf
         Ref: app/site.someones.Lonewolf/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 1.0.4
     License: GPL-3.0-only
      Origin: flathub
  Collection: org.flathub.Stable
Installation: system
   Installed: 147,0 MB
     Runtime: org.gnome.Platform/x86_64/45
         Sdk: org.gnome.Sdk/x86_64/45

      Commit: 557d21501ea4efaafad49090e9375472811aba280bd055fff87e74ddbfa9b3e2
      Parent: baa01e05d5ae28a3149fe530ea74556a77973373cd42b94aa5f1e362247baa84
     Subject: Bump version (1.0.4) (82c8a2a3)
        Date: 2023-10-05 19:09:51 +0000
```
```

terkyz@terkyz-linux:~$ 

terkyz@terkyz-linux:~$ flatpak info --show-permissions site.someones.Lonewolf
[Context]
shared=network;ipc;
sockets=x11;wayland;fallback-x11;
devices=dri;
filesystems=home;
```

```
terkyz@terkyz-linux:~$ flatpak run site.someones.Lonewolf
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1000x600: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1000x600: Permission denied
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1000x600: Permission denied
Failed to create EGL images for DMABufs with file descriptors -1, -1 and -1

```

I'm unable to give output of the app's console because it not shown anything even if I press on "Inspect Element".

**Comment by TurbooGhost** [comment]
Yup same here!

**Comment by aichingm** [comment]
Thanks for the data. I checked the `KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied` line which seams to be an issue with webkit https://groups.google.com/g/linux.debian.bugs.dist/c/jqFSXiFozd4.

Could you try executing?
```
flatpak override --env=WEBKIT_DISABLE_DMABUF_RENDERER=1 site.someones.Lonewolf
flatpak run site.someones.Lonewolf
```
Does this work/change the output?

**Comment by TurbooGhost** [comment]
Still the same output. Cannot reload or click on "Inspect Element"

**Comment by aichingm** [comment]
@TurbooGhost @Terkyz could you test if this build works?

I set up a new Debian 12 system but can't reproduce the issue... My KDE neon install works too. 

```
flatpak install https://dl.flathub.org/build-repo/55128/site.someones.Lonewolf
```

**Comment by TurbooGhost** [comment]
<pre><code>sudo flatpak install https://dl.flathub.org/build-repo/55128/site.someones.Lonewolf
Looking for matches…
F: An error was encountered searching remote ‘flathub’ for ‘https://dl.flathub.org/build-repo/55128/site.someones.Lonewolf’: 'https:' is not a valid name: Name can't contain :
error: No remote refs found for ‘https://dl.flathub.org/build-repo/55128/site.someones.Lonewolf’</code></pre>

**Comment by TurbooGhost** [comment]
Hello @aichingm 

I recently cloned your [Repo Name] repository and tried to get it up and running. However, I encountered a few issues and have some suggestions:

- The `tsconfig` configurations seem to reference certain paths and types that aren't available or are misconfigured. This leads to TypeScript compilation errors.
- I noticed there's some confusion regarding where the Vite configurations should reside, whether in `lonewolf-web` or `lonewolf-tauri`.
- Certain essential dependencies like Vite seem to be missing or not installed correctly. I had to manually install them to proceed.
- The hardcoded local server URL (`http://localhost:5173/`) might pose challenges in some development scenarios or deployment. It would be good to make this configurable.

I'm interested in contributing to the project and helping address these issues. Would it be possible for me to contribute directly to the repo or should I fork it and submit pull requests? 

Looking forward to collaborating!

**Comment by Terkyz** [comment]
> flatpak run site.someones.Lonewolf

In my case, this solves the problem.

**Comment by TurbooGhost** [comment]
It worked @aichingm @Terkyz 

Thanks guys!

**Comment by TurbooGhost** [comment]
Yup using nvidia gpu.

Sorry for that! It just works perfectly! Thank you very much. You may close the issue now if you would like.

**Comment by Terkyz** [comment]
> FYI this is the bug report at Webkit https://bugs.webkit.org/show_bug.cgi?id=259644
> 
> I will release a version with webkit 2.40.5
> 
> @TurbooGhost @Terkyz Just to check are you both using nvidia gpus?
> 
> @TurbooGhost can you open a separate issue for your four point so that we can discuss them separately. Pull Requests are very very welcome 😃

Yep, I also use a Nvidia GPU.

**Comment by aichingm** [comment]
I updated Lonewolf to a new major version of Tauri which uses a newer version of webkitgtk which should not have this problems anymore. Should be released in 2.0.0.

Can be tested in this test build:
```
flatpak install --user https://dl.flathub.org/build-repo/184139/site.someones.Lonewolf.flatpakref
```

Feel free to reopen if necessary.

**Comment by ssteinerx** [comment]
That URL 404s

**Comment by aichingm** [comment]
Yes the test builds are only available for a limited amount of time...

**Comment by mimbasher** [comment]
Hey @aichingm, the issue is now resolved. The app is working much better and launches significantly faster, thanks to Tauri! 🙌 Huge thanks for building this awesome app. I’ll be contributing very soon with some cross-platform improvements and small fixes here and there. Excited to be part of this project!
